Cannot define upload type in swagger file

I am trying to define a file upload file

```
/upload:
     post:
       summary: Uploads a CSV file.
       consumes:
         - multipart/form-data
       parameters:
         - in: formData
           name: customers
           type: string
           format: binary
           description: The file to upload.
         - name: update
           in: query
           type: boolean
           required: false
           default: false
         - name: import_id
           in: query
           type: string
           required: false
```

But it results in broken code gen

```
restapi/operations/post_upload_parameters.go:131:14: cannot use value.(*io.ReadCloser) (type *io.ReadCloser) as type io.ReadCloser in assignment:
	*io.ReadCloser is pointer to interface, not interface
```

--- a/templates/parameter.gotmpl
+++ b/templates/parameter.gotmpl
@@ -244,7 +244,7 @@ func ({{ .ReceiverName }} *{{ pascalize .Name }}Params) readRequest(ctx *gin.Con
   if err != nil {
     res = append(res, errors.New(400, "reading file %q failed: %v", {{ printf "%q" (camelize .Name) }}, err))
   } else {
-    {{ .ReceiverName }}.{{ pascalize .Name }} = runtime.File{Data: {{ camelize .Name }}, Header: {{ camelize .Name }}Header}
+    {{ .ReceiverName }}.{{ pascalize .Name }} = &runtime.File{Data: {{ camelize .Name }}, Header: {{ camelize .Name }}Header}
   }
   {{ else }}fd{{ pascalize .Name }}, fdhk{{ pascalize .Name }}, _ := fds.GetOK({{ .Path }})
   if err := {{ .ReceiverName }}.bind{{ pascalize .ID }}(fd{{ pascalize .Name }}, fdhk{{ pascalize .Name }}, formats); err != nil {

